rust/BUILD.md

```diff
@@ -0,0 +1,134 @@
+# `evmrs` Development Instructions
+
+## Requirements
+
+- install Rust toolchain >= 1.81.0 from [here](https://rustup.rs/)
+
+## Building evmrs (shared c library)
+
+- build `evmrs` in release mode
+    ```sh
+    cargo build --release # OR run `make` or `make tosca-rust` in Tosca directory
+    ```
+- build `evmrs` with debug symbols
+    ```sh
+    cargo build --profile profiling
+    ```
+
+## Lint
+
+```sh
+cargo clippy --workspace --tests --benches --examples
+```
+
+## Doc
+
+```sh
+cargo doc --workspace --document-private-items --open
+```
+
+## Optimizations
+
+By default, `evmrs` is build with the simplest and most idiomatic version.
```

Comment:
version of what?  this sentence is a little loose.

Reply:
... most idiomatic implementation?